src/lib.rs

```diff
@@ -212,6 +212,17 @@ impl CommitDomain {
         }
     }
 
+    /// Constructs a new `CommitDomain` from different values for `hash_domain` and `blind_domain`
+    pub fn new_with_personalization(hash_domain: &str, blind_domain: &str) -> Self {
```

Comment:
```suggestion
    pub fn new_with_separate_domains(hash_domain: &str, blind_domain: &str) -> Self {
```
(`CommitDomain::new` is always personalized.)

Reply:
Done